bookkeeper-server/src/main/java/org/apache/bookkeeper/util/PortManager.java

```diff
@@ -54,21 +52,14 @@ public static void initPort(int initPort) {
     public static synchronized int nextFreePort() {
         int exceptionCount = 0;
         while (true) {
-            int port = nextPort++;
```

Comment:
should we drop "nextPort" ?

Reply:
Yes I think so. I've added a commit to remove it and all the initialPort arguments that would end up setting it.